### PR TITLE
Don't log error when blob was already deleted by another instance of a job

### PR DESCRIPTION
**Pull request by rkondratowicz**
The main job is not using schedlock, locks on files are used instead.
Currently, when a job tries to acquire a lease for a file that is already being process it just silently skips it.

Adding the same logic for when blob was already deleted.

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/LeaseAcquirer.java [hunk 1]
```diff
@@ -31,7 +33,7 @@ public void ifAcquiredOrElse(BlobClient blobClient, Runnable onSuccess, Runnable
             release(leaseClient, blobClient);
 
         } catch (BlobStorageException exc) {
-            if (exc.getErrorCode() != BlobErrorCode.LEASE_ALREADY_PRESENT) {
+            if (exc.getErrorCode() != LEASE_ALREADY_PRESENT && exc.getErrorCode() != BLOB_NOT_FOUND) {
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/LeaseAcquirer.java:36]: in else may be we can log as warn

Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/LeaseAcquirer.java:36]: should the condition be OR ?

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/LeaseAcquirer.java:36]: with OR it would always be true :)

**Comment by rkondratowicz** [comment]
jenkins issues, closing

**Pull request closed without merging**